packages/util/src/error-with-cause.ts

```diff
@@ -15,6 +20,11 @@ export class ErrorWithCause extends Error {
     Object.setPrototypeOf(this, new.target.prototype); // restore prototype chain
     this.name = 'ErrorWithCause';
 
+    // Axios removed the stack property in version 0.27 which gave no useful information anyway. This adds the http cause.
+    if (isAxiosError(cause)) {
+      this.stack = `${this.stack}${unixEOL}Caused by:${unixEOL}Http Response: ${cause.message} - ${cause?.response?.data}`;
+    }
+
     // Stack is a non-standard property according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#Custom_Error_Types
     if (this.stack && cause?.stack) {
```

Comment:
Don't you need an else here to make sure that there is never a "Caused by: ... Caused by: ..."? (In case the axios error by chance has a `.stack`)

Reply:
hm, the axiosError does not have a stack anymore. But I guess it is saver to return or use an else.